privaterelay/tests/mgmt_update_phone_remaining_stats_tests.py

```diff
@@ -266,3 +267,40 @@ def test_phone_subscriber_with_subscription_end_date_sooner_than_31_days_since_r
     assert relay_number.remaining_texts == settings.MAX_TEXTS_PER_BILLING_CYCLE
     assert relay_number.remaining_minutes == settings.MAX_MINUTES_PER_BILLING_CYCLE
     assert relay_number.remaining_seconds == settings.MAX_MINUTES_PER_BILLING_CYCLE * 60
+
+
+def test_phone_subscriber_with_subscription_end_date_after_reset_phone_limits_updated(
+    phone_user,
+):
+    datetime_now = datetime.now(timezone.utc)
+    datetime_first_of_march = datetime_now.replace(month=3, day=4)
+    profile = Profile.objects.get(user=phone_user)
+    profile.date_subscribed_phone = datetime_now.replace(month=1, day=2)
+    new_subscription_start_and_previous_reset_date = datetime_now.replace(
+        month=2, day=1
+    )
+    profile.date_phone_subscription_start = (
+        new_subscription_start_and_previous_reset_date
+    )
+    profile.date_phone_subscription_end = datetime_first_of_march
+    profile.date_phone_subscription_reset = (
+        new_subscription_start_and_previous_reset_date
+    )
+    profile.save()
+    relay_number = _make_used_relay_number(phone_user)
+
+    # today must be after march 3rd, to use the calculated reset date (subscription end date >= calculated_next_reset_date)
```

Comment:
(comment) Good use of comments here. This means the test will fail if it is being ran before March 3rd (Jan 1 - Mar 2nd) which is a bug for a test that is being ran 24/7/365 as part of the CI/CD process.